README.rst

```diff
@@ -208,6 +208,8 @@ You can refresh the Docker hosted page at http://localhost:4080/figure to see ch
     $ cd /PATH_TO_GIT_REPO/omero-figure
     $ grunt build
     $ grunt watch   # to see updates
+
+    # in a different terminal...
```

Comment:
In the cmd below you are using the OMEROHOST variable which was defined in the previous terminal. Maybe move the OMEROHOST definition to here instead ?

Reply:
@pwalczysko instead, I switched the order you launch these so that `grunt` is in a different terminal and you launch docker first in the terminal that does have OMEROHOST - see a6940e16